### PR TITLE
youtube-music: 3.1.0 -> 3.2.2, little cleanup

### PR DESCRIPTION
**Pull request by SuperSandro2000**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
@ofborg build youtube-music

File: pkgs/applications/audio/youtube-music/default.nix [hunk 1]
```diff
@@ -51,17 +46,15 @@ stdenv.mkDerivation (finalAttrs: {
...
     dontFixup = true;
     outputHashMode = "recursive";
-    outputHashAlgo = "sha256";
     outputHash = {
-      x86_64-linux = "sha256-Oy11V7FXfVhLUR9gX0sjQEFuVPFpbaVdT518oOSLcvA=";
-      aarch64-linux = "sha256-6nXemaGiQjp2stjjKItPJ62VcH5Q5pRf63qKtl2haXI=";
+      x86_64-linux = "sha256-/C6S2WhS8J8sNlOCdEZRRuW6Q3gAOGOlAQXTVvK1Ln0=";
+      aarch64-linux = "sha256-qgXfkfwr1kcVJvzhisFHv7qriOqvNxQXAbNS9ZTCr+E=";
       x86_64-darwin = "sha256-jSMAw+AMD63vqPckZjblw4EDngA4E8h0WlsZu3hUShY=";
       aarch64-darwin = "sha256-zujXURpIcw7IOw63AW167h6cywYXydhHZMzA2apGZAs=";
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/youtube-music/default.nix:53]:
```suggestion
      x86_64-linux = "sha256-SOtueFJ+w6k9BQMbyd7xtbQ4CusExIEO5gMRywu4+0I=";
      aarch64-linux = "sha256-MHnVoQMqDvCqNpmV/lFKGlGJErXZEKPjdWfgEUszu90=";
      x86_64-darwin = "sha256-XK7qof8HQ3qf7Dwql04+0yWvHOPI5YanfID+y1VuM+Y=";
      aarch64-darwin = "sha256-zujXURpIcw7IOw63AW167h6cywYXydhHZMzA2apGZAs=";
```

**Review by aacebedo — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
I am a bit confused why ofborg is suggesting other things than when building locally despite running multiple rebuilds.

File: pkgs/applications/audio/youtube-music/default.nix [hunk 2]
```diff
@@ -51,17 +46,15 @@ stdenv.mkDerivation (finalAttrs: {
...
     outputHashMode = "recursive";
-    outputHashAlgo = "sha256";
     outputHash = {
-      x86_64-linux = "sha256-Oy11V7FXfVhLUR9gX0sjQEFuVPFpbaVdT518oOSLcvA=";
-      aarch64-linux = "sha256-6nXemaGiQjp2stjjKItPJ62VcH5Q5pRf63qKtl2haXI=";
-      x86_64-darwin = "sha256-jSMAw+AMD63vqPckZjblw4EDngA4E8h0WlsZu3hUShY=";
+      x86_64-linux = "sha256-SOtueFJ+w6k9BQMbyd7xtbQ4CusExIEO5gMRywu4+0I=";
+      aarch64-linux = "sha256-MHnVoQMqDvCqNpmV/lFKGlGJErXZEKPjdWfgEUszu90=";
+      x86_64-darwin = "sha256-XK7qof8HQ3qf7Dwql04+0yWvHOPI5YanfID+y1VuM+Y=";
       aarch64-darwin = "sha256-zujXURpIcw7IOw63AW167h6cywYXydhHZMzA2apGZAs=";
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/youtube-music/default.nix:53]: If someone could provide that it would be awesome. ofborg is taking it's sweet time

Comment (SuperSandro2000) [reply pkgs/applications/audio/youtube-music/default.nix:53]:
```suggestion
      aarch64-darwin = "sha256-VrQdKrji/U3xjRfav87voUEisewukVRB4zAUBW4HOIw=";
```

File: pkgs/applications/audio/youtube-music/default.nix [hunk 3]
```diff
@@ -51,17 +46,15 @@ stdenv.mkDerivation (finalAttrs: {
...
     dontFixup = true;
     outputHashMode = "recursive";
-    outputHashAlgo = "sha256";
     outputHash = {
-      x86_64-linux = "sha256-Oy11V7FXfVhLUR9gX0sjQEFuVPFpbaVdT518oOSLcvA=";
-      aarch64-linux = "sha256-6nXemaGiQjp2stjjKItPJ62VcH5Q5pRf63qKtl2haXI=";
-      x86_64-darwin = "sha256-jSMAw+AMD63vqPckZjblw4EDngA4E8h0WlsZu3hUShY=";
-      aarch64-darwin = "sha256-zujXURpIcw7IOw63AW167h6cywYXydhHZMzA2apGZAs=";
+      x86_64-linux = "sha256-V6CSawxBWFbXmAPbck0xCXqRlANpqFAoqSAB4Duf8qM=";
+      aarch64-linux = "sha256-MHnVoQMqDvCqNpmV/lFKGlGJErXZEKPjdWfgEUszu91=";
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/youtube-music/default.nix:51]:
```suggestion
      aarch64-linux = "sha256-cqBn35soV14CmobKt0napRELio4HKKA8Iw3QSWTxzP8=";
```

File: pkgs/applications/audio/youtube-music/default.nix [hunk 4]
```diff
@@ -51,17 +46,15 @@ stdenv.mkDerivation (finalAttrs: {
...
-    outputHashAlgo = "sha256";
     outputHash = {
-      x86_64-linux = "sha256-Oy11V7FXfVhLUR9gX0sjQEFuVPFpbaVdT518oOSLcvA=";
-      aarch64-linux = "sha256-6nXemaGiQjp2stjjKItPJ62VcH5Q5pRf63qKtl2haXI=";
-      x86_64-darwin = "sha256-jSMAw+AMD63vqPckZjblw4EDngA4E8h0WlsZu3hUShY=";
-      aarch64-darwin = "sha256-zujXURpIcw7IOw63AW167h6cywYXydhHZMzA2apGZAs=";
+      x86_64-linux = "sha256-V6CSawxBWFbXmAPbck0xCXqRlANpqFAoqSAB4Duf8qM=";
+      aarch64-linux = "sha256-cqBn35soV14CmobKt0napRELio4HKKA8Iw3QSWTxzP8=";
+      x86_64-darwin = "sha256-XK7qof8HQ4qf7Dwql04+0yWvHOPI5YanfID+y1VuM+Y=";
+      aarch64-darwin = "sha256-VrQdKrji/U4xjRfav87voUEisewukVRB4zAUBW4HOIw=";
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/youtube-music/default.nix:53]:
```suggestion
      x86_64-darwin = "sha256-DY9T1N8Hxr57/XisYT+u2+hQvYMIiyQ3UHeTuA6BhSY=";
      aarch64-darwin = "sha256-VrQdKrji/U4xjRfav87voUEisewukVRB4zAUBW4HOIw=";
```

**Pull request merged**